Feature/quick sankey cards

jesusgonzalezrivera opened the pull request:
Sankey card links entity has the `level` field to represent the three levels available through the design. There is a maximum number of sankey card links for each level, so:
- `level = 1` - This level is used when the user has not selected commodity and country. It permits a maximum of 4 Sankey card links.
- `level = 2` - When the user has selected the commodity. It permits a maximum of 4 Sankey card links for each commodity.
- `level = 3` - When the user has selected commodity and country (context). It permits a maximum of 4 Sankey card links for each context.

agnessa commented on app/admin/api/v3/sankey_card_link.rb:
```diff
@@ -0,0 +1,53 @@
+      link_to(sankey_card_link.link, sankey_card_link.link)
+    end
+    column :level do |sankey_card_link|
+      [1, 2, 3].select { |n| sankey_card_link.send("level#{n}") }.join(', ')
```
I think it would be better to avoid "magic numbers" in favour of a constant, e.g. `Api::V3::SankeyCardLink::LEVELS = [1,2,3].freeze` and use that whenever we need to enumerate the levels

jesusgonzalezrivera commented:
You are completely right. I have fixed the code with the comments you have written and I have also added the logic for updating `query_params` when any of the relations changes.
